swan-lake/integration-tools/openapi-tool.md

```diff
@@ -71,17 +97,43 @@ The command-line arguments below can be used with the command for each particula
 | `--single-file`           | This option can be used to generate the Ballerina service or client with related types and utility functions in a single file. | Optional |
 | `--use-sanitized-oas`     | This is an experimental feature. This option enables service/client code generation by modifying the given OAS to follow Ballerina language best practices. | Optional |
 
-### Ballerina to OpenAPI command options
+### Ballerina to OpenAPI
 
 The command-line arguments below can be used with the command for each particular purpose as described below.
 
 | Command option    | Description                                                                                                                                                                                               | Mandatory/Optional |
 |-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|
-| `-i \|--input`    | The `ballerina-service-file-path` command option specifies the path of the Ballerina service file (e.g., `my-service.bal`).                                                                               | Mandatory          |
+| `-i \|--input`    | This specifies the path of the Ballerina service file (e.g., `my-service.bal`).                                                                               | Mandatory          |
 | `--json`          | Generate the Ballerina service to OpenAPI output as JSON. The default is YAML.                                                                                                                            | Optional           |
 | `-s \| --service` | This service name is used to identify the service that needs to be documented as an OpenAPI contract.                                                                                                | Optional           |
 | `-o\|--output`    | Location of the generated OpenAPI contract. If this path is not specified, the output is written to the same directory from which the command is run.                                                | Optional           |
 
+### Sanitize OpenAPI
+
+The command-line arguments below can be used with the command for each particular purpose as described below.
```

Comment:
Still don't think is necessary. Can just say something along the lines of the following options are available.